pkg/ddc/jindocache/transform.go

```diff
@@ -453,13 +453,13 @@ func (e *JindoCacheEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, m
 			key := encryptOption.Name
 			secretKeyRef := encryptOption.ValueFrom.SecretKeyRef
 			if secretMountSupport {
-				value.Secret = secretKeyRef.Name
+				value.Secret = secretKeyRef.Key
```

Comment:
I think value.Secret is secretKeyRef.Name not secretKeyRef.Key. Because secretKeyRef.Name  is the name of secret object.